cmd/snap-repair/runner_test.go

```diff
@@ -1881,18 +1881,21 @@ var _ = Suite(&runScriptSuite{})
 
 func (s *runScriptSuite) SetUpTest(c *C) {
 	s.baseRunnerSuite.SetUpTest(c)
+	s.runDir = filepath.Join(dirs.SnapRepairRunDir, "canonical", "1")
+
+	s.AddCleanup(snapdenv.SetUserAgentFromVersion("1", nil, "snap-repair"))
+
+	restoreErrTrackerReportRepair := repair.MockErrtrackerReportRepair(s.errtrackerReportRepair)
+	s.AddCleanup(restoreErrTrackerReportRepair)
+}
 
+func (s *runScriptSuite) setup(c *C) {
```

Comment:
(nitpick) I would call this `setupRunner()` and we probably also want a comment why this can't be part of `SetUpTest()`.

Reply:
Good point, I'll change that. Thanks